apis/nucleus/src/components/listbox/interactions/keyboard-navigation/keyboard-nav-rows.js

```diff
@@ -114,7 +114,7 @@ export default function getRowsKeyboardNavigation({
         break;
       case KEYS.ENTER:
         confirm();
-        break;
+        return;
```

Comment:
would
```
if (isModal()) {
  confirm();
} else {
  return; // propagate to other handler
}
```
work?
I think it would be safer and more correct for the case when the listbox is not a popover.

I think the change it OK for the filterpane case anyway as enter will probably not do anything it should not.
but I would be good to test to verify it.

Reply:
@T-Wizard Yes it works with a bit of modification. 
isModel() is almost always undefined in ListBoxPopover so the check statement throw a type error (isModel is not a function), we get the same error when using Esc after making selections(That I had mentioned before). EnterKey should confirm the selections in all listboxes so I should only ensure that if the listbox is not popover, it will stop propagation and keeps the list open.